view/frontend/web/js/view/payment/method-renderer/adyen-google-pay-method.js

```diff
@@ -150,6 +154,22 @@ define(
                     self.googlePayAllowed(false);
                 });
             },
+            /**
+             * Based on the response we can start a 3DS2 validation or place the order
+             * @param responseJSON
+             */
+            validateThreeDSOrPlaceOrder: function (responseJSON, orderId) {
+                var response = JSON.parse(responseJSON);
+                var googlepayRedirectUrl = '';
+                if (response.type === 'RedirectShopper') {
+                    googlepayRedirectUrl = window.checkoutConfig.payment[quote.paymentMethod().method].redirectUrl
+                } else {
```

Comment:
did you test the refused non3DS1 flow and the refused 3DS1 flow as well?

Reply:
Tests:
- Success 3DS 1 Google pay
- Success non 3DS 1 Google pay
- Refused 3DS 1 Google pay (wrong password)

The refused non3DS1 was not possible to test it as clicking back on the google pay pop up returns to checkout screen

Reply:
perfect, thanks :)